setup.py

```diff
@@ -115,7 +115,8 @@ def run_tests(self):
         'pyyaml',
         'argparse',
         'argcomplete',
-        'future'
+        'future',
+        'pyBarcode==0.8b1'
```

Comment:
I am not sure about the steenzout-fork being community-unfriendly, but the only benefit of this fork seem to be some unit tests and a CLI. Unit tests can also be added and we don't need a CLI.

Because of the lacking ambiguity I would also tend to use viivakoodi. I think it is best to wait if the project is still alive and then you can change the dependecy here.